### DISCUSSION TITLE
Tuning jpg quality to save disk space

### DISCUSSION THREAD

**Discussion by nipodemus** [aaronwmorris/indi-allsky#1532] [Q&A] [answered]
Hi, I am running indi-allsky with RPI4 and RPI HQ camera. It is working well. Amazing tool, thank you!
Is there a for example libcamera option available to tune jpg quality? Mainly I would like to have day timelapses but with lower jpg quality to save disk space.

**Comment by quanticleap** [comment]
I have pi hq camera and using binning value "2" for Night Bin Mode and Daytime Bin Mode in menu Config-Camera , also for timelapse I'm using config-image-FFMPEG Scaling 25% or v760 px and this save space on sd in my case

**Comment by aaronwmorris** [comment, marked as answer]
There is not a way to separate day and night for the JPEG compression, but the setting for JPEGs is under the Image tab at `JPEG Quality` (about half way down).  The default is 90.

**Reply by aaronwmorris** [reply]
Also, one of the changes I made recently to the defaults is only saving 10 days of images.  It will not change the existing setting (it was only for new installs).  But saving 10 days vs 30 will save a lot of space.

**Reply by nipodemus** [reply]
Thank you!
